test/e2e/preempt.go

```diff
@@ -0,0 +1,206 @@
+package e2e
+
+import (
+	"context"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	corev1 "k8s.io/api/core/v1"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	schedulingv1beta1 "volcano.sh/volcano/pkg/apis/scheduling/v1beta1"
+)
+
+var _ = Describe("Job E2E Test", func() {
+	It("schedule high priority job without preemption when resource is enough", func() {
+		ctx := initTestContext(options{
+			priorityClasses: map[string]int32{
+				masterPriority: masterPriorityValue,
+				workerPriority: workerPriorityValue,
```

Comment:
suggest fix them in a separate pr, maybe call high/lowPriority

Reply:
OK, i will fix the name in next pr

Reply:
it has been fixed at https://github.com/volcano-sh/volcano/pull/893